app-emulation/fs-uae/fs_uae-2.8.3.recipe

```diff
@@ -0,0 +1,119 @@
+SUMMARY="FS-UAE is a cross platform Amiga emulator ported from WinUAE"
+DESCRIPTION="Accurate Amiga Emulation
+
+FS-UAE integrates the most accurate Amiga emulation code available from \
+WinUAE. FS-UAE emulates A500, A500+, A600, A1200, A1000, A3000 and A4000 \
+models, but you can tweak the hardware configuration and create customized \
+Amigas.
+
+On-Screen GUI With Gamepad Control
+
+FS-UAE is fully controllable with a gamepad - you can easily swap floppies \
+and load save states through the on-screen GUI. In addition to being great \
+on your personal computer, FS-UAE is also well suited to be started from an \
+emulator frontend running on a HTPC.
+
+Configuration GUI
+
+FS-UAE includes an easy-to-use graphical configuration program which allows \
+you to enjoy FS-UAE and Amiga emulation without writing configuration files. \
+Using this program is optional, and if you are a minimalist, you can instead \
+use just FS-UAE alone.
+
+Available for Multiple Operating Systems
+
+FS-UAE works on Windows, Linux, Mac OS X, FreeBSD and OpenBSD. Apart from a \
+few small differences, the feature set is the same on all platforms. The same \
+configurations, floppy images and hard drives can be shared between multiple \
+systems.
+
+Emulate High-End Amigas
+
+FS-UAE supports emulating an Amiga 4000 with an CPU as fast as your \
+system can emulate it. Picasso96 drivers can be used with the built-in \
+UAEgfx card to display Workbench in high resolutions and with up to 16.7 \
+million colors, and the built-in bsdsocket.library means that the Amiga \
+will have access to the Internet.
+
+Community Support
+
+There is a support forum for FS-UAE on the English Amiga Board with many \
+active registered users. If you are stuck, you'll almost certainly get an \
+answer to your questions there. You can also leave comments on fs-uae.net \
+is you rather want that.
+
+Online Documentation
+
+A great deal of documentation about how to use and configure FS-UAE is \
+available on fs-uae.net."
+HOMEPAGE="https://fs-uae.net/"
+COPYRIGHT="2011-2017 Frode Solheim"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://fs-uae.net/stable/${portVersion}/fs-uae-${portVersion}.tar.gz"
+CHECKSUM_SHA256="e2d5414d07c8bd5b740716471183bc5516bec0ae2989449c3026374dc4b86292"
+SOURCE_DIR="fs-uae-$portVersion"
+PATCHES="fs_uae-$portVersion.patchset"
+
+ARCHITECTURES="?x86 !x86_gcc2 x86_64"
+
+PROVIDES="
+	fs_uae = $portVersion
+	cmd:fs_uae
+	cmd:fs_uae_device_helper
+	add_on:fs_uae.dat
```

Comment:
Absolutely, yes :) It's needed for all the images and other resources. I added support to load it from the right location. It tries to load it from the current working directory, so you wouldn't have noticed if you ran fs-uae in the build area.